volume/service/service.go

```diff
@@ -60,18 +60,23 @@ func (s *VolumesService) GetDriverList() []string {
 	return s.ds.GetDriverList()
 }
 
+// AnonymousLabel is the label used to indicate that a volume is anonymous
+// This is set automatically on a volume when a volume is created without a name specified, and as such an id is generated for it.
+const AnonymousLabel = "com.docker.volume.anonymous"
```

Comment:
One thing I wonder; will this have an effect on volumes that "started as anonymous", but later on became "named"?

Thinking of situations like

```
docker run -d --name container1 some-image-with-volumes

docker run --volumes-from=container2 some-image
```

Yes, the volume is still "anonymous", but no longer really "ephemeral"

Reply:
The volume is still anonymous, but if a container is holding a reference to it, it won't be pruned anyway.

Reply:
Yeah, makes sense